lib/tasks/update_application_form_status.rake

```diff
@@ -0,0 +1,7 @@
+namespace :db do
+  desc "Update application status"
+  task update_status: :environment do
```

Comment:
```suggestion
  task update_application_form_status_initial_assessment: :environment do
```

Although it's a temporary task, I think it's still worth giving it a clearer name. Worth making the filename match the task too I think.